crates/types/src/camera_position.rs

```diff
@@ -1,5 +1,11 @@
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+#[derive(
+    Default, Debug, Clone, Copy, Eq, PartialEq, Serialize, Deserialize, SerializeHierarchy,
+)]
 pub enum CameraPosition {
+    #[default]
```

Comment:
Is this required?

Reply:
Yes, because this type is now used in the generated configuration structs which require `Default`.